src/twisted/conch/scripts/ckeygen.py

```diff
@@ -23,13 +24,25 @@
 
 
 
+supportedKeyTypes = dict()
+def _keyGenerator(keyType):
+    def assignkeygenerator(keygenerator):
+        supportedKeyTypes[keyType] = keygenerator
+        @wraps(keygenerator)
+        def wrapper(*args, **kwargs):
+            keygenerator(*args, **kwargs)
+        return wrapper
```

Comment:
Sorry, this isn't quite right and that's why you're short on coverage. `@wraps` should go on `assignKeyGenerator`, not `wrapper`, and you shouldn't need `wrapper` at all.